boards/nxp/imx95_evk/imx95_evk_mimx9596_a55_smp.dts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mimx95_a55.dtsi>
+#include "imx95_evk-pinctrl.dtsi"
+
+/ {
+	model = "NXP i.MX95 A55";
+	compatible = "fsl,mimx95";
+
+	chosen {
+		zephyr,console = &lpuart1;
+		zephyr,shell-uart = &lpuart1;
+		zephyr,sram = &sram0;
+	};
```

Comment:
SMP runs on all A55 Core? suggest SMP variants runs on two CPU Core, so that it can demo how to select SMP runs on which CPU Core.

And also update board document for introduction of how SMP variants works and how to run it.

Reply:
I think currently we'd better add the SMP support with all A55 cores, and use the U-Boot 'go' command to let the boot core bring up the secondary cores, as the 'cpu' command in U-Boot cmd/mp.c has been deprecated in upstream, which is used to kick the secondary cores, and the new 'cpu' command in cmd/cpu.c is still not supporting release secondary cores from reset. I'm planning to add the 'release' subcommand to the new 'cpu' command, and will update the document when it is accepted.

Reply:
okay, thanks.